package.json

```diff
@@ -83,7 +83,8 @@
     "dev": "rimraf ./lib && babel -w src --out-dir lib --copy-files",
     "lint": "eslint '{src,test}/**/*.js'",
     "plaintest": "mocha --exit -t 10000 -b -R spec test/index.js && npm run tape",
-    "prepublish": "npm run babel",
+    "prepare": "npm run babel",
```

Comment:
why was this changed?

Reply:
This sounds out of scope for this PR, though. Could you prepare a separate PR with explanation of the change for this/